client/app/src/state/actions.js

```diff
@@ -1,10 +1,12 @@
 import concertApi from '../api/concertApi';
 import { CHOICE_CHILLS } from '../constants/Choices';
 import { CHOICE_MADE } from '../constants/Events';
+import { AMPLIFY_AUTH_DISABLE } from 'react-native-dotenv'; //to check if MTURK version
```

Comment:
```suggestion
import { AMPLIFY_AUTH_DISABLE } from 'react-native-dotenv';
```
This is not a useful comment, it's duplicated below